workflowparsers/lobster/metainfo/lobster.py

```diff
@@ -79,6 +83,167 @@ class Method(runschema.method.Method):
         """,
     )
 
+class x_lobster_section_cobi(MSection):
+    """
+    This is a section containing the crystal orbital bond index (COBI)
+    and integrated cobi (iCOBI) values.
+    """
+
+    m_def = Section(validate=False)
+
+    x_lobster_number_of_cobi_pairs = Quantity(
+        type=int,
+        description="""
+        Number of atom pairs for which are the COBIs and iCOBIs calculated.
+        """,
+    )
+
+    x_lobster_cobi_atom1_labels = Quantity(
+        type=str,
+        shape=["x_lobster_number_of_cobi_pairs"],
+        description="""
+        Species and indices of the first atom for which is the specific COBI/iCOBI calculated
+        """,
+    )
+
+    x_lobster_cobi_atom2_labels = Quantity(
+        type=str,
+        shape=["x_lobster_number_of_cobi_pairs"],
+        description="""
+        Species and indices of the second atom for which is the specific COBI/iCOBI calculated
+        """,
+    )
+
+    x_lobster_cobi_distances = Quantity(
+        type=np.dtype(np.float64),
```

Comment:
nowadays you can use directly `np.float64` or whatever type, without the need of `np.dtype`

Reply:
Done